File: Arrays/sumTarget.js
const twoSum = (nums, target)=>{
    let newArr=new Map();
    for(let i=0; i<nums.length; i++) {
        let resta= target - nums[i];
        if(newArr.has(resta)){
            console.log("====",newArr[resta])
            return [i, newArr.get(resta)]
        }
        newArr.set(nums[i],i);
        console.log(newArr)

    }
};
console.log(twoSum([4,3,1,2],6))

File: Arrays/sumTarget.py
class Solution(object):
    def twoSum(self, nums, target):
        hashTable={}
        print('a')
        for index, num in enumerate(nums):
            resta=target-num
            if resta in hashTable:
                print('++',[hashTable[resta],index] )
                return [hashTable[resta],index] 
            hashTable[num]=index
            print('--',hashTable)

        return []